packages/twenty-server/src/engine/core-modules/auth/services/sign-in-up.service.ts

```diff
@@ -229,6 +231,8 @@ export class SignInUpService {
         email,
       );
 
+      await this.userService.markEmailAsVerified(updatedUser.id);
```

Comment:
Right now we don't force the user who signs up with an invitation to use the same email their were invited on so technically we can't be 100% sure that email is verified, unless we force signing up with the same email as in the invitation. I'm not sure if it's worth doing though...

Reply:
We actually do enforce that: 

https://github.com/twentyhq/twenty/blob/b22a598d7d1901b940ad91be47358f696d31abf9/packages/twenty-server/src/engine/core-modules/workspace-invitation/services/workspace-invitation.service.ts#L95

I verified by sending an invite to `a@a.com` and changing the sign up email to `b@b.com`. We get back an error:

![image](https://github.com/user-attachments/assets/5d89836b-723e-458c-a623-8c2e088e3672)

Reply:
@FelixMalfait Are we good resolving this one?

Reply:
Yes for private invitations we enforce the email. For me it's ok.

Reply:
Sounds good, resolving this conversation.